kai_solution_server/service/incident_store/incident_store.py

```diff
@@ -528,6 +528,9 @@ def incident_store_from_config(config: KaiConfig) -> "IncidentStore":
         KAI_LOG.info(f"Selected provider: {config.models.provider}")
         KAI_LOG.info(f"Selected model: {model_provider.model_id}")
 
+        if config.incident_store is None:
+            raise ValueError
+
```

Comment:
This currently raises a `type[ValueError]` not a `ValueError`. I think you want to raise `ValueError()` instead.